SonarQube.TeamBuild.PostProcessor/MSBuildPostProcessor.cs

```diff
@@ -60,13 +64,6 @@ public bool Execute(string[] args, AnalysisConfig config, TeamBuildSettings sett
                 return false;
             }
 
-            if (config == null)
-            {
-                LogStartupSettings(config, settings, logger);
-                logger.LogError(Resources.ERROR_MissingSettings);
-                return false;
-            }
-
```

Comment:
Miscellaneous change: the missing config has already been logged by the calling Program.cs - no point in logging it twice.
ERROR_MissingSettings is not user. I'll remove it.